### DISCUSSION TITLE
Static local variable colorization

### DISCUSSION THREAD

**Discussion by AlexanderNarbekov** [microsoft/vscode-cpptools#10284] [Q&A]
Is there a way to colorize a function's variable that has  the "local" modifier but also the "static" storage modifier? ( i mean tokenColorCustomizationse's stuff)
Unfortunately that variable has only one modifier (i understand because "static" refers to the keyword, not a variable) and I don't know how to customize a scope for my purpose. 
Thanks.

**Comment by sean-mcmanus** [comment]
I don't know. @Colengms might know.

**Reply by sean-mcmanus** [reply]
This sounds unrelated to our extension, so you could also try VS Code's feedback channels: https://stackoverflow.com/questions/tagged/visual-studio-code and https://github.com/microsoft/vscode-discussions/discussions/5 .

**Reply by AlexanderNarbekov** [reply]
Thanks for reply! I try to ask there.
P.S.
I had read up this [https://code.visualstudio.com/docs/cpp/colorization-cpp](url) topic and I was thought C/C++ extension resolves Token Names
